runtime/lua/vim/_editor.lua

```diff
@@ -627,6 +627,113 @@ function vim.notify(msg, level, opts) -- luacheck: no unused args
   end
 end
```

Comment:
nit: it's a bit random that the code was added in between vim.notify and vim.notify_once :)